src/BigAww.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component, PropTypes } from 'react';
+
+class BigAww extends Component {
```

Comment:
check out React pure functional components. This doesnt actually need to be a whole class. Mind blown when I learned about them